nautobot/dcim/tests/test_views.py

```diff
@@ -1887,10 +1887,10 @@ def setUpTestData(cls):
         }
 
         cls.csv_data = (
-            "device,name,type,status",
-            f"{device.composite_key},Interface 4,1000base-t,{statuses[0].name}",
-            f"{device.pk},Interface 5,1000base-t,{statuses[0].name}",
-            f"{device.composite_key},Interface 6,1000base-t,{statuses[0].name}",
+            "type,name,device__name,device__location__name,status",
+            f"virtual,Interface 4,{device.name},{device.location.name},{statuses[0].name}",
+            f"1000base-t,Interface 5,{device.name},{device.location.name},{statuses[0].name}",
+            f"1000base-t,Interface 6,{device.name},{device.location.name},{statuses[1].name}",
```

Comment:
This looks good as a spot test but might be good to add a baseline test in `nautobot.core.tests.test_api.NautobotCSVParserTest` as well.